editors/code/src/installation/download_file.ts

```diff
@@ -1,6 +1,13 @@
 import fetch from "node-fetch";
 import * as fs from "fs";
+import * as stream from "stream";
+import * as util from "util";
 import { strict as assert } from "assert";
+import { NestedError } from "ts-nested-error";
+
+const pipeline = util.promisify(stream.pipeline);
+
+class DownloadFileError extends NestedError {}
```

Comment:
We never catch this specific error, so I don't think it makes much sense to do error wrapping at all.

Reply:
This is just that the name of the error (i.e. the name of the class reflectively) is displayed in the error message. Though we can infer where the Errror came from by the nested callstacks, so this is not that important...

Reply:
Yeah, I would prefer to keep things simple by default, or rely on ubiquitous libraries (like rust's anynow) if there are big gains.

Reply:
Sure, will simplify!